src/external/spglib.jl

```diff
@@ -44,21 +44,27 @@ spglib_cell(lattice, atoms) = first(spglib_cell_atommapping_(lattice, atoms))
     end
 
     # Ask spglib for symmetry operations and for irreducible mesh
-    spg_symops = spglib.get_symmetry(spglib_cell(lattice, atoms),
-                                     symprec=tol_symmetry)
+    spg_rotations    = Array{Cint}(undef, 3, 3, max_symops)
+    spg_translations = Array{Cdouble}(undef, 3, max_symops)
+    spg_positions, spg_numbers = spglib_atoms(atoms)
+    
+    spg_num_ops = ccall((:spg_get_symmetry, SPGLIB), Cint,
+       (Ptr{Cint}, Ptr{Cdouble}, Cint, Ptr{Cdouble}, Ptr{Cdouble}, Ptr{Cint}, Cint, Cdouble),
+       spg_rotations, spg_translations, Cint(max_symops), lattice, spg_positions, spg_numbers,
+       Cint(length(spg_numbers)), tol_symmetry)
+       
+    # Note: spglib_jll returns transposed rotation matrix vs the python spglib interface
```

Comment:
We don't reference the python interface anymore so this isn't helpful; rather at the top of the file I'd write down what spglib's convention is